bot/exts/utils/reminders.py

```diff
@@ -537,18 +570,32 @@ async def _can_modify(self, ctx: Context, reminder_id: str | int, send_on_denial
                 if e.status == 404:
                     return False
             raise e
+        owner_id = api_response["author"]
 
-        if await has_any_role_check(ctx, Roles.admins):
+        if owner_id == ctx.author.id:
+            log.debug(f"{ctx.author} is the reminder's author and passes the check.")
             return True
 
-        if api_response["author"] != ctx.author.id:
-            log.debug(f"{ctx.author} is not the reminder author and does not pass the check.")
-            if send_on_denial:
-                await send_denial(ctx, "You can't modify reminders of other users!")
-            return False
+        if await has_any_role_check(ctx, Roles.admins):
+            log.debug(f"{ctx.author} is an admin, asking for confirmation to modify someone else's.")
 
-        log.debug(f"{ctx.author} is the reminder author and passes the check.")
-        return True
+            confirmation_view = ModifyReminderConfirmationView(ctx.author)
+            await ctx.reply(
+                "Are you sure you want to modify <@{owner_id}>'s reminder?",
```

Comment:
This currently says `Are you sure you want to modify {{mention}}'s reminder?` when you're trying to delete it, which isn't quite explicit and doesnt point the invoker to the action.

It's better to make this clearer, and IIRC you'll have access to that from `ctx.command.invoked_subcommand`

Reply:
Maybe the action being performed could be passed into the function instead? Although it might be redundant since a Context object is already being passed in.

I think I will stick with using `ctx.command.invoked_subcommand` for now.

Reply:
Also, would it be worth changing the message in L597 to also include the modification type? I feel like doing so might imply that editing/deleting other people's reminders have different permissions when they actually don't.

Reply:
> Also, would it be worth changing the message in L597 to also include the modification type? I feel like doing so might imply that editing/deleting other people's reminders have different permissions when they actually don't.

No you don't need to, as that message is destined to people who don't have the rights to do anything with a reminder.

The reason I asked you to make it explicit for adminds is because it will pinpoint the action they're about to do better.